### DISCUSSION TITLE
[IntelliJ IDEA]No completions were found

### DISCUSSION THREAD

**Discussion by Zer0ON1** [community/community#17658] [Copilot Conversations]
无论我输入什么它都提示No conpletions were found
IDEA版本号2022.1.2
copilot版本号1.1.16.1219

whatever I type,it says No completions were found
My IDEA version is 2022.1.2 copilot version is 1.1.16.1219

p.s. when update to 1.1.20 or higher version,it even cannot link Github to login
<img width="1280" alt="42bd6d76b258567f545503c604a0308" src="https://user-images.githubusercontent.com/104151831/171581033-710eb397-cc41-4589-9c9c-39945a2e7117.png">

**Comment by jansorg** [comment]
Thanks for reporting! Are you using a proxy server?

**Reply by Zer0ON1** [reply]
Negative

**Reply by LazyB0705** [reply]
我也是这样 你已经解决了吗
IDEA版本2021.1.2
copilot版本1.1.20.1417
用DevSidecar改了系统代理

**Reply by longzheng268** [reply]
我也是，我是Clion。

**Reply by 1che34** [reply]
我也是。。

**Reply by zainzhoucom** [reply]
+1

**Reply by AaronFelix1991** [reply]
+1

**Reply by huntswyj** [reply]
same

**Reply by lyk4411** [reply]
+1
